src/helpers/tests/measureTitleHelper.spec.ts

```diff
@@ -205,4 +206,97 @@ describe("measureTitleHelper", () => {
             ).toBeUndefined();
         });
     });
+
+    describe("ignoreTitlesForSimpleMeasures", () => {
+        function getTitleOfMeasure(
+            visualizationObject: IVisualizationObjectContent,
+            localIdentifier: string,
+        ): string {
+            const measureBucketItems: BucketItem[] = visualizationObject.buckets[0].items;
+            const matchingMeasure: IMeasure = measureBucketItems
+                .map((bucketItem: BucketItem): IMeasure => bucketItem as IMeasure)
+                .find(
+                    (bucketItem: IMeasure): boolean => bucketItem.measure.localIdentifier === localIdentifier,
+                );
+
+            return matchingMeasure === undefined ? undefined : matchingMeasure.measure.title;
+        }
+
+        function getAliasOfMeasure(
+            visualizationObject: IVisualizationObjectContent,
+            localIdentifier: string,
+        ): string {
+            const measureBucketItems: BucketItem[] = visualizationObject.buckets[0].items;
+            const matchingMeasure: IMeasure = measureBucketItems
+                .map((bucketItem: BucketItem): IMeasure => bucketItem as IMeasure)
+                .find(
+                    (bucketItem: IMeasure): boolean => bucketItem.measure.localIdentifier === localIdentifier,
+                );
+
+            return matchingMeasure === undefined ? undefined : matchingMeasure.measure.alias;
+        }
```

Comment:
these functions are similar, please try to make them reuseable :)

Reply:
Fixed!